examples/src/main/scala/sttp/tapir/examples/security/csrfTokens.scala

```diff
@@ -0,0 +1,186 @@
+// {cat=Security; effects=Future; server=Pekko HTTP}: Securing endpoint with CSRF tokens example
```

Comment:
maybe let's do this one with Netty-sync, to have some more variety in the interpreters

Reply:
done

Reply:
you need to change the `server` here then :)